pkg/koyeb/utils.go

```diff
@@ -69,3 +72,29 @@ func loadYaml(file string) (string, error) {
 	}
 	return "", errors.New("Unknown format")
 }
+
+func watchDeployment(h *ServiceHandler, deploymentId string) {
+	numAttemptsOnUnhealthy := 12
+	retryCount := 0
+	retryInterval := 5
+
+	for {
+		res, resp, err := h.client.DeploymentsApi.GetDeployment(h.ctx, deploymentId).Execute()
+		if err != nil {
+			fatalApiError(err, resp)
+		}
+		currentStatus := res.Deployment.GetStatus()
+
+		log.Infof("Service deployment in progress. Deployment status is %s. Next update in %d seconds.", currentStatus, retryInterval)
```

Comment:
This works fine for a first version, but maybe we should only print that in debug mode.
We could also introduce later a 'spinner' to replace this, and this message in debug mode.